src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.runtimeconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "runtimeOptions": {
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "1.0.0-rc3-002733"
```

Comment:
Are these runtimeconfig.json files strictly required by the shared host? That design seems problematic in cases like this, where we just want to try to run on whatever shared framework is available (we don't really care).